wiki/osu!_File_Formats/Osu_(file_format)/en.md

```diff
@@ -60,22 +60,30 @@ LetterboxInBreaks (Boolean) specifies whether the letterbox appears during break
 
 `LetterboxInBreaks: 0`
 
-WidescreenStoryboard (Boolean) specifies whether or not the storyboard should be widescreen.
-
-`WidescreenStoryboard: 0`
-
 StoryFireInFront (Boolean) specifies whether or not display the storyboard in front of combo fire.
 
 `StoryFireInFront: 0`
 
-SpecialStyle (Boolean) specifies whether or not use the special `N+1` style for osu!mania.
+SkinPreference (String) specified the preferred skin to use during gameplay.
```

Comment:
```diff
- specified
+ specifies
```

Reply:
Oops my bad. How do I add the typo fix in? New to Github